psycop/common/feature_generation/sequences/timeseries_windower_python/prediction_time.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import datetime as dt
+    from collections.abc import Sequence
+
+    from psycop.common.feature_generation.sequences.timeseries_windower_python.events.static_feature import (
+        StaticFeature,
+    )
+    from psycop.common.feature_generation.sequences.timeseries_windower_python.events.temporal_event import (
+        TemporalEvent,
+    )
+    from psycop.common.feature_generation.sequences.timeseries_windower_python.patient import (
+        Patient,
+    )
```

Comment:
We have talked about this one, but why not have the structure:

```
from psycop.data_structures import Patient, Event, StaticFeature  
```

and have the location of the scripts be:
```
from psycop.data_structures.patient import Patient  
```
Even if we don't want to maintain init files I would much rather have this last one.